### ISSUE TITLE
[Usage] pretrain about "gradient_accumulation_steps" para use

### ISSUE THREAD

**Issue by wanlipeng** [haotian-liu/LLaVA#1670]
### Describe the issue

When I pre-train, the parameter gradient_accumulation_steps is set to 1, and the total training step length is 23256. However, when I set gradient_accumulation_steps to 4, the total training step length is 23256/4=5814. Will the gradient accumulation parameter affect the total training step length? Is it different from the usual torch training?
